assets/prototype/application/services/config/ConfigData.ts

```diff
@@ -0,0 +1,85 @@
+import {
+    Currency,
+    CurrencyProps,
+    CurrentUser,
+    CurrentUserProps,
+    DateTimeFormats,
+    DateTimeFormatsProps,
+    Locale,
+    LocaleProps,
+    SiteUrl,
+    SiteUrlProps,
+    Timezone,
+    TimezoneProps,
+} from '../../valueObjects/config';
+
+declare global {
+    interface Window {
+        eejsdata: eeJsProps;
+    }
+}
+
+window.eejsdata = window.eejsdata || {} as eeJsProps;
+const data: eeJsDataProps = window.eejsdata.data || {} as eeJsDataProps;
```

Comment:
Smth optional:
in ES2020 optional chaining and nullish coalescing operator will be available
As of TS 3.7 we can use both today:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing
***
https://levelup.gitconnected.com/new-features-of-javascript-that-we-can-use-soon-or-now-6199981bd2f

Reply:
In this context we might simplify code by doing this:
`const data: eeJsDataProps = window?.eejsdata?.data ?? {}`

Reply:
I don't think this is hype or smth, in some cases, it really makes code a little bit cleaner.

Reply:
As I commented at the beginning: this is optional.
If this proposal will be ignored - I am ok with that :)

Reply:
oh VERY cool, thnx for showing me that 👍